HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -857,48 +753,16 @@ class HyprMXAdapter : PartnerAdapter {
             )
         }
 
-        override fun onAdDisplayError(
-            placement: Placement?,
-            hyprMXError: HyprMXErrors?,
-        ) {
+        override fun onAdDisplayError(placement: Placement, hyprMXError: HyprMXErrors) {
```

Comment:
did they move to a kotlin sdk or something? that's why we're able to move to not `?`?